### PR TITLE
f/100019_News_Feed

### PR DESCRIPTION
**Pull request by Janaki-Venkatesh**
#### What's this PR do?
  this is for viewing news reagarding cognizant studio book
#### Where should the reviewer start?
         NewsFeed.js
#### How should this be manually tested?
         After login click this link http://localhost:3000/dashboard to view newsfeed
#### Any background context you want to provide?
         no context
#### New dependencies? What are they used for?
         "node-sass-chokidar": "^1.3.3",
    "npm-run-all": "^4.1.3",
    "primeicons": "^1.0.0-beta.10",
    "primereact": "^2.0.0-beta.1",
    "react": "^16.4.1",
    "react-dom": "^16.4.1",
    "react-md": "^1.4.2",
    "react-redux": "^5.0.7",
    "react-router-dom": "^4.3.1",
    "react-router-redux": "^4.0.8",
    "react-scripts": "1.1.4",
    "redux": "^4.0.0",
    "redux-thunk": "^2.3.0",
    "webfontloader": "^1.6.28"
#### Who did you pair with on this change?
          No  one
#### What are the relevant Pivotal Tracker issues?
         Nothing
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/container/NewsFeedPage.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import {TabView,TabPanel} from 'primereact/tabview';
+import { Badge,  SVGIcon } from 'react-md';
+
+class NewsFeedPage extends React.Component {
+  
+    
+    
+    constructor() {
+       super();
+
```
Comment (ghost) [inline src/container/NewsFeedPage.js:20]: remove line space here

**Review by ghost — CHANGES_REQUESTED** [review]
reduce the tabs

File: src/component/Login.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+             </div>
+             <div className="md-grid md-cell-10">
+             <TextField  id="floating-password"label="Enter your password" type="password" />
+             </div>
+             <div className="md-grid md-cell-10">
+             <Button onClick={() =>handleLogin()} flat primary swapTheming>Login</Button>
+             </div>
+            </CardText>
+       </Card>
+
```
Comment (Ashok-R-Mohan) [inline src/component/Login.js:25]: remove white space

**Review by Ashok-R-Mohan — CHANGES_REQUESTED** [review]
make the mentioned changes

**Pull request merged**